### ISSUE TITLE
How can I train the DR-DSNsup model?

### ISSUE THREAD

**Issue by Wwwwei** [KaiyangZhou/pytorch-vsumm-reinforce#18]
In the paper, you mention how to train the DSNsup. But there is so little information about how to train the DR-DSNsup, can you specify it?

Thank you.

**Comment by SinDongHwan** [comment]
this code not support supervised learning.

**Comment by Wwwwei** [comment]
hello, I just want to know how to combine the supervised learning with DR-DSN. Does it means the object function is the sum of cross-entropy loss and DR reward function ？
I just want to try it on my own.
Thanks.

**Comment by SinDongHwan** [comment]
I think so. 
I will try to do it.

**Comment by SinDongHwan** [comment]
Hello, @Wwwwei . 
I think so.
I will try to do it.

> hello @SinDongHwan, I just want to know how to combine the supervised learning with DR-DSN. Does it means the object function is the sum of cross-entropy loss and DR reward function ？
> I just want to try it on my own.
> Thanks.
> 
> > this code not support supervised learning.
> > Duplicate of #7

**Comment by LeoLiu1220** [comment]
did u finish it ? i think the cross-entropy loss is more less more better and the reward function is the opposite 
so how did u do
